### PR TITLE
created Message format class

### PR DESCRIPTION
**Pull request by jonsals**

### CODE REVIEW

**Review by koder23 — APPROVED** [review]

File: MessageFormat.java [hunk 1]
```diff
@@ -0,0 +1,73 @@
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.List;
+
+
+public class MessageFormat implements Serializable{
+    public int id;
+    private int numFields;
+    private String ipAdd;
+    private int port;
```
Comment (AdrianSLopez) [inline MessageFormat.java:10]: hardcode this

File: MessageFormat.java [hunk 2]
```diff
@@ -0,0 +1,73 @@
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.List;
+
+
+public class MessageFormat implements Serializable{
+    public int id;
+    private int numFields;
+    private String ipAdd;
```
Comment (AdrianSLopez) [inline MessageFormat.java:9]: hardcode this, get ip from Constants

File: MessageFormat.java [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+        return id;
+    }
+    public void setId(int id){
+        this.id = id;
+    }
+
+    public String getIpAddress(){
+        return ipAdd;
+    }
+    public void setIpAddress(String ipAdd) {
```
Comment (AdrianSLopez) [inline MessageFormat.java:34]: Since IP will be hard coded remove this

File: MessageFormat.java [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+    }
+    public void setNumFields(int numFields){
+        this.numFields = numFields;
+    }
+
+    
+    public int getPort(){
+        return port;
+    }
+    public void setPort(int port){
```
Comment (AdrianSLopez) [inline MessageFormat.java:50]: Remove setPort since Port will also be hardcoded

File: MessageFormat.java [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+
+    public List<String>getRoutingTable(){
+        return routingTable;
+    }
+
+
+    public void setRoutingTabel(List<String> routingTable){
+        this.routingTable = routingTable;
+    }
+
```
Comment (AdrianSLopez) [inline MessageFormat.java:72]: Can you add a method to add the Message's routing table, will help with the update command.

File: MessageFormat.java [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+import java.util.ArrayList;
+import java.util.List;
+
+
+public class MessageFormat implements Serializable{
+    public int id;
+    private int numFields;
+    private String ipAdd;
+    private int port; 
+    private List<String> routingTable = new ArrayList<String>();
```
Comment (AdrianSLopez) [inline MessageFormat.java:11]: Change name to ServerUpdates.

**Pull request merged**